### PR TITLE
fix: update centos docker to 2.2.

### PR DESCRIPTION
**Pull request by moonming**
fix https://github.com/apache/apisix-docker/issues/116

### CODE REVIEW

File: centos/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-ARG APISIX_VERSION=1.2
+ARG APISIX_VERSION=2.2
 LABEL apisix_version="${APISIX_VERSION}"
 
 RUN yum -y install yum-utils\
 	&& yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo \
 	&& yum install -y openresty \
-	&& yum install -y https://github.com/apache/apisix/releases/download/$APISIX_VERSION/apisix-$APISIX_VERSION-0.el7.noarch.rpm \
+	&& yum install -y https://github.com/apache/apisix/releases/download/$APISIX_VERSION/apisix-$APISIX_VERSION-0.x86_64.rpm \
```
Comment (nic-chen) [inline centos/Dockerfile:9]: only 2.2 uses `x86_64`. maybe we should make it compatible with older versions?

Comment (moonming) [reply centos/Dockerfile:9]: We can modify the names of all apisix rpm packages

Comment (moonming) [reply centos/Dockerfile:9]: all keep the name `x86_64`

**Review by spacewander — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

File: centos/Dockerfile (hunk 1 above)
Comment (membphis) [reply centos/Dockerfile:9]: yes, that is a better way

**Pull request merged**